src/registry_schemas/example_data/schema_data.py

```diff
@@ -107,9 +107,9 @@
         },
         {
             'role': 'amalgamating',
+            'identifier': '123456',
             'legalName': 'Foreign Co.',
-            'foreignJurisdiction': FOREIGN_JURISDICTION,
-            'corpNumber': '123456'
```

Comment:
A bunch of code will need to be updated in Create UI if we change corpNumber to identifier. I can do it but I want it to be visible work. Can you call this out in standup tomorrow, please?

Also, is there any plan to correct the existing amalgamation filings to match this?

Reply:
Yes. I am planning to created separate ticket to update these. I can create a query to update json and rename the db field

Reply:
OK, sounds good.

Reply:
https://app.zenhub.com/workspaces/entities---olga-65af15f59e89f5043c2911f7/issues/gh/bcgov/entity/19561